src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {

```

Comment:
In the constructor, there should be checks for null values and inappropriate data for the parameters.